Lib/multiprocessing/util.py

```diff
@@ -102,6 +102,29 @@ def log_to_stderr(level=None):
     _log_to_stderr = True
     return _logger
 
+
+# Abstract socket support
+
+def _platform_supports_abstract_sockets():
+    if sys.platform == "linux":
+        return True
+    if hasattr(sys, 'getandroidapilevel'):
+        return True
+    return False
+
+
+def is_abstract_socket_namespace(address):
+    if not address:
```

Comment:
This code path accepts an empty tuple. But I don't know if it's an issue.

Reply:
Is not because an empty tuple cannot be an abstract socket namespace address.

Reply:
Ok, that's fine.